models/user.go

```diff
@@ -20,6 +21,21 @@ const (
 	usersTable string = "users"
 )
 
+//Authenticates the user's credentials and returns the user
+func (db *DB) CheckUser(user *User) (*User, error) {
+	userFromDB := &User{}
+	queryString := fmt.Sprintf("SELECT ID, Name, Password FROM %s WHERE EMAIL=?", usersTable)
+	if err := db.QueryRow(queryString, user.Email).Scan(&(userFromDB.ID),
```

Comment:
Might be more readable to break this up into two lines, like:
```
err := db.QueryRow()
if err != nil {
```

Reply:
oh true, I'll fix that rn

Reply:
done